### PR TITLE
Adds minor fixes to wikidatawiki features and rebuilds model.

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

**Comment by halfak** [comment]
Built this on ores-compute-01

```
ScikitLearnClassifier
 - type: RF
 - params: bootstrap=true, center=true, class_weight=null, scale=true, max_depth=null, n_jobs=1, min_weight_fraction_leaf=0.0, oob_score=false, criterion="gini", min_samples_split=2, random_state=null, warm_start=false, min_samples_leaf=13, max_leaf_nodes=null, balanced_sample=true, n_estimators=20, verbose=0, max_features="log2", balanced_sample_weight=false
 - version: 0.1.0
 - trained: 2017-06-29T21:46:08.457256

Table:
	      ~A    ~B    ~C    ~D    ~E
	--  ----  ----  ----  ----  ----
	A    295    20     7     0     0
	B     35   344    54     4     1
	C     55   102  1588    24     3
	D      0     1    33   930    33
	E      0     0     3    27  1437

Accuracy: 0.92
ROC-AUC:
	---  -----
	'A'  0.987
	'B'  0.962
	'C'  0.982
	'D'  0.98
	'E'  0.994
	---  -----

F1:
	-  -----
	B  0.76
	E  0.977
	A  0.834
	C  0.919
	D  0.939
	-  -----
```

File: wikiclass/feature_lists/wikidatawiki.py [hunk 1]
```diff
@@ -76,9 +76,13 @@ def simplify_target(target):
         return target
     elif hasattr(target, "toTimestr"):
         return target.toTimestr()
+    elif isinstance(target, dict) and 'text' in target:
```
Comment (adamwight) [inline wikiclass/feature_lists/wikidatawiki.py]: These switch clauses could use some comments, your reader is left wondering why `target` is such a slippery variable.

Comment (halfak) [inline wikiclass/feature_lists/wikidatawiki.py]: This is a fair point.  I should really say "WTF wikidata/pywikibase none of this is documented!"

**Comment by Pix1234** [comment]
docs are lacking in general. i think its time to go on a doc updating
spree? @halfak

Zppix
Volunteer developer for WMF
enwp.org/User:Zppix

**Comment by halfak** [comment]
pywikibase isn't something that we officially support.  I'm working on an alternative that strips out all of the pywikibot-ish stuff from it.  See [mwbase](https://github.com/mediawiki-utilities/python-mwbase).  I've been working on [an update to revscoring that will allow us to switch to that entirely](https://github.com/wiki-ai/revscoring/commit/b3c863af49f2d2419fc0c69bb656a4e415d997c8).  Once that is ready, I agree that docs should be better.

If you like drama, see also https://phabricator.wikimedia.org/T108440

**Pull request merged**